usr/local/www/diag_resetstate.php

```diff
@@ -43,21 +43,31 @@
 ##|*MATCH=diag_resetstate.php*
 ##|-PRIV
 
+define("DEBUG", true);	// Displays "success" messages, but does not call the functions that reset the firewall state tables
```

Comment:
I understand the intention, but I'm not sure it should be part of this branch :)